sync/syncer_test.go

```diff
@@ -72,7 +72,7 @@ func TestSyncer_Start(t *testing.T) {
 	syncs, _ := SyncMockFactory(2, conf, "TestSyncer_Start_")
```

Comment:
Id like for us to write random tests, if you want you can define what a random UT would look like and we'll let a contributor do it